### PR TITLE
added ewok and IReligious

### PR DESCRIPTION
**Pull request by JasonWeakley**
Added the Ewok and IReligious

### CODE REVIEW

File: Galaxy Wars/Galaxy Wars/Species/Ewok.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      this.health += 1;
+      this.stealth += 8;
+      this.rage += 2;
+      this.luck += 7;
+      this.communication += 3;
+    }
+
+    public void beginEncryptBonus()
+    {
+      this.encryptBonusActive = true;
```
Comment (franticallycodes) [inline Galaxy Wars/Galaxy Wars/Species/Ewok.cs]: This should change to attackBonusActive instead.

File: Galaxy Wars/Galaxy Wars/Species/Ewok.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      this.communication -= 5;
+    }
+
+    public Population ReligiousBonus(Population galaxyPopulation)
+    {
+      galaxyPopulation.ewokPopulation += galaxyPopulation.saiyanPopulation * 0.01;
+      galaxyPopulation.saiyanPopulation -= galaxyPopulation.saiyanPopulation * 0.01;
+      galaxyPopulation.ewokPopulation += galaxyPopulation.wookiePopulation * 0.01;
+      galaxyPopulation.wookiePopulation -= galaxyPopulation.wookiePopulation * 0.01;
+    }
```
Comment (franticallycodes) [inline Galaxy Wars/Galaxy Wars/Species/Ewok.cs:42]: This method should return galaxyPopulation.

**Comment by franticallycodes** [comment]
:+1:

**Comment by craftylildev** [comment]
:+1:

**Pull request merged**